.github/workflows/guide.yml

```diff
@@ -55,7 +55,7 @@ jobs:
       # Only deploy guide when releasing a new version of Hermes
       - name: Deploy to GitHub Pages
         uses: peaceiris/actions-gh-pages@v3
-        if: github.event_name == 'push' && startsWith(github.ref, 'refs/tags/v0.')
+        if: github.event_name == 'push' && startsWith(github.ref, 'refs/tags/v0')
```

Comment:
Still not v1-ready :grin:

Reply:
hehe

Reply:
Hopefully we'll have a better solution for the guide by then, ie. with versioning etc.

Reply:
Maybe if you rely on the workflow trigger being either push to `master` or to a `v[0-9]+\.*` tag, it's enough to discern here that the push was for a tag.

Reply:
Good point!